src/core/Form/Textarea/Textarea.baseStyles.tsx

```diff
@@ -5,13 +5,11 @@ import { element, font } from '../../theme/reset';
 export const baseStyles = (theme: SuomifiTheme) => css`
   ${element(theme)}
   ${font(theme)('bodyText')}
-
-    &.fi-textarea {
-    display: flex;
-    flex-direction: column;
-    color: ${theme.colors.blackBase};
-    width: 290px;
-
+  display: flex;
```

Comment:
If the purpose of this change is to make the styles more easy to override, do we really want this of the `display` value, since that's quite crucial for many components and a different value very likely breaks the structure of the component?